seatunnel-ci-tools/src/test/java/org/apache/seatunnel/api/ConnectorOptionCheckTest.java

```diff
@@ -0,0 +1,233 @@
+                                                                                            "Options"));
+                                                                }
+                                                            });
+                                                    NodeList<ClassOrInterfaceType> extendedTypes =
+                                                            classDeclaration.getExtendedTypes();
+                                                    extendedTypes.forEach(
+                                                            extendedType -> {
+                                                                if (extendedType
+                                                                                .getNameAsString()
+                                                                                .equals(
+                                                                                        "AbstractSimpleSink")
+                                                                        || extendedType
+                                                                                .getNameAsString()
+                                                                                .equals(
+                                                                                        "AbstractSingleSplitSource")) {
+                                                                    connectorOptionFileNames.add(
+                                                                            path.getFileName()
+                                                                                    .toString()
+                                                                                    .replace(
+                                                                                            JAVA_FILE_EXTENSION,
+                                                                                            "")
+                                                                                    .concat(
+                                                                                            "Options"));
+                                                                }
+                                                            });
+                                                }
+                                            });
+                        } catch (IOException e) {
+                            throw new RuntimeException(e);
+                        }
+                    });
+            connectorClassPaths.forEach(
+                    path -> {
+                        String className =
+                                path.getFileName().toString().replace(JAVA_FILE_EXTENSION, "");
+                        connectorOptionFileNames.remove(className);
+                    });
+            connectorOptionFileNames.removeAll(buildWhiteList());
```

Comment:
we should assert `connectorOptionFileNames` size equals white list size.

Reply:
It seems that this might not be necessary.
If the `white list size` is smaller than `connectorOptionFileNames.size()`, the following `Assertions.assertEquals(0, connectorOptionFileNames.size())` will fail.

On the other hand, if the `white list size` is larger than `connectorOptionFileNames.size()`, removing non-existent `Options` or already removed `Options` won't have any impact.

Additionally, if we check for equality of size before calling `connectorOptionFileNames.remove(className)`, the size will never match when the update PR is done and the connector class is removed from the white list.

Please correct me if I'm wrong.

Reply:
Sorry, my logic needs to be corrected. We should check the `size of connectorOptionFileNames be removed by white list` equals `white list size`. So that we can make sure the `white list` are all up to date and right.